src/styles/main.scss

```diff
@@ -1,3 +1,73 @@
-body {
+$cell-size: 100px;
+$field-padding: 10px;
+$base-shift: 1px;
+$calendar-width: $cell-size * 7 + $field-padding + $base-shift;
+$weekdays: 
+  "mon" 0, 
+  "tue" 1, 
+  "wed" 2, 
+  "thu" 3, 
+  "fri" 4, 
+  "sat" 5, 
+  "sun" 6;
+
+* {
   margin: 0;
+  padding: 0;
```

Comment:
do not reset every possible margin/padding via `*` selector, it could be damaging